### PR TITLE
[DBA.dk] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

File: src/chrome/content/rules/DBA.dk.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		- til-boligen	(m)
+		- toej-og-mode	(m)
+		- tsgw	(t)
+		- tv	(m)
+		- ww	(m)
+		- www01	(m)
+		- www03	(m)
+		- www08	(m)
+		- www1	(m)
+		- wwww	(m)
```
Comment (J0WI) [inline src/chrome/content/rules/DBA.dk.xml]: Wildcard DNS.

**Comment by youdly** [comment]
Updated comment

File: src/chrome/content/rules/DBA.dk.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+	Host has wildcard DNS
+-->
+<ruleset name="DBA.dk">
+
+	<target host="dba.dk" />
+	<target host="www.dba.dk" />
+	<target host="admanager.dba.dk" />
+	<target host="api.dba.dk" />
+	<target host="autodiscover.dba.dk" />
```
Comment (J0WI) [inline src/chrome/content/rules/DBA.dk.xml]: Does not work anymore

**Comment by youdly** [comment]
Removed `autodiscover`. Rebased.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**